src/main/java/com/rovio/ingest/WriterContext.java

```diff
@@ -97,7 +102,12 @@ private WriterContext(CaseInsensitiveStringMap options, String version) {
         this.s3BaseKey = options.getOrDefault(ConfKeys.DEEP_STORAGE_S3_BASE_KEY, null);
         this.s3DisableAcl = options.getBoolean(ConfKeys.DEEP_STORAGE_S3_DISABLE_ACL, false);
         this.localDir = options.getOrDefault(ConfKeys.DEEP_STORAGE_LOCAL_DIRECTORY, null);
-        this.getHdfsStorageDir = options.getOrDefault(ConfKeys.DEEP_STORAGE_HDFS_DIRECTORY, null);
+        this.hdfsDir = options.getOrDefault(ConfKeys.DEEP_STORAGE_HDFS_STORAGE_DIRECTORY, null);
+        this.hdfsCoreSitePath = options.getOrDefault(ConfKeys.DEEP_STORAGE_HDFS_CORE_SITE_PATH, null);
+        this.hdfsHdfsSitePath = options.getOrDefault(ConfKeys.DEEP_STORAGE_HDFS_HDFS_SITE_PATH, null);
+        this.hdfsDefaultFS = options.getOrDefault(ConfKeys.DEEP_STORAGE_HDFS_DEFAULT_FS, null);
```

Comment:
These 3 conf options are left hidden – no mention in README.
* DEEP_STORAGE_HDFS_DEFAULT_FS has a "hidden" implementation
* X_SITE_PATHs are not implemented (exception is thrown if these are set).

This is how it was in https://github.com/fabricebaranski/rovio-ingest/commit/9c6eb40e1b7b00ac9f5126e38bb27c8e644ef173. We can keep them as a starting point for possible implementation in the future.